sisl/viz/plots/bands.py

```diff
@@ -3,21 +3,24 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 from collections import defaultdict
 from functools import partial
+import importlib
 import itertools
 
 import numpy as np
+import xarray as xr
 
 import sisl
 from ..plot import Plot, entry_point
-from ..plotutils import call_method_if_present, find_files
+from ..plotutils import find_files
 from ..input_fields import (
-    TextInput, SwitchInput, ColorPicker, DropdownInput,
-    IntegerInput, FloatInput, RangeInput, RangeSlider,
-    QueriesInput, ProgramaticInput, FunctionInput, SileInput,
-    PlotableInput, SpinSelect, AiidaNodeInput, BandStructureInput
+    TextInput, SwitchInput, ColorPicker,
+    FloatInput, RangeSlider,
+    QueriesInput, FunctionInput, SileInput,
+    SpinSelect, AiidaNodeInput, BandStructureInput
 )
 from ..input_fields.range import ErangeInput
 
+_do_parallel_calc = importlib.util.find_spec("pathos") is not None
```

Comment:
why not just `try ... except`? using an entire module for this seems not needed?

Reply:
perhaps the `try except` could carry a warning for threaded blas/lapack such that users are aware of potential risks? This could even be checked against OMP_NUM_THREADS == 1 (in which case no warning is required).

Reply:
The `import` statement already uses `importlib`, so I don't think this adds much overhead, no? Anyway, I'll change it if you prefer it.

Reply:
I would prefer try except, it is basically what you do, so no need to complicate things. True about the existing overhead anyways!